pkg/reconciler/pipelinerun/pipelinerun_test.go

```diff
@@ -6453,33 +6513,93 @@ func (prt PipelineRunTest) reconcileRun(namespace, pipelineRunName string, wantE
 	return reconciledRun, clients
 }
 
+func TestReconcile_RemotePipelineRefValidation_Failure(t *testing.T) {
+	names.TestingSeed()
```

Comment:
nit: I don't think this line is necessary

Reply:
fixed